Python/omevv/v1/scripts/CreateBaselineProfile.py

```diff
@@ -0,0 +1,206 @@
+        self.baseline_profileid = ""
+        self.payload = {}
+
+
+    #<--- Function to get user inputs--->
+    def create_payload(self,name,description,firmware_repoid,driver_repoid,time,retry,configuration_repoid=None,createdby=None,group_idslist=[],jobschedule_days="Sunday"):
+        self.retry = retry
+
+        self.payload = {
+            "name": name,
+            "description": description,
+            "firmwareRepoId": firmware_repoid,
+            "driverRepoId": driver_repoid,
+            "configurationRepoId": configuration_repoid,
+            "createdBy": createdby,
+            "groupIds": group_idslist,
+            "jobSchedule": {
+                "monday": False,
+                "tuesday": False,
+                "wednesday": False,
+                "thursday": False,
+                "friday": False,
+                "saturday": False,
+                "sunday": True,
+                "time": time
+            }
+        }
+
+        #jobscheduledays
+        if jobschedule_days:
+            #When multiple days are mentioned
+            if "," in jobschedule_days:
+                split_string = jobschedule_days.split(",")
+                for day in split_string:
+                    if day in self.payload["jobSchedule"]:
+                        self.payload["jobSchedule"][day]= True
+                    else:
+                        print("\n FAIL, either missing argument or incorrect argument used. Please check script help-text for jsd.")
+                        print("\n Proceeding with default value for jobschedule-day")
+            else:
+                if self.payload["jobSchedule"].get(jobschedule_days):
+                    self.payload["jobSchedule"][jobschedule_days]= True
+                else:
+                    print("\n INFO, incorrect argument value for jobschedule-day. Please check script help-text for jsd.")
+                    print("\n Proceeding with default value for jobschedule-day")
+
+        #Removing the items in payload with None values
+        self.payload = {k: v for k, v in self.payload.items() if v}
+
+        print("\n Payload is: {}".format(self.payload))
+        return self.payload
+
+
+
+    #<--- Function to  create_baselineprofile--->
+    def create_baselineprofile(self,payload,vc_username,vc_pwd,omevv_ip,vc_uuid,retry):
+
+        self.vc_username=vc_username
+        self.vc_pwd=vc_pwd
+        self.omevv_ip=omevv_ip
+        self.vc_uuid=vc_uuid
+
+        try:
+            credential = self.vc_username + ":" + self.vc_pwd
+            self.encoded_vc_cred = "Basic %s" % base64.b64encode(credential.encode('utf-8')).decode()
+            self.headers["Authorization"] = self.encoded_vc_cred
+            self.headers.update({"x_omivv-api-vcenter-identifier": vc_uuid})
+
+            self.config_url = "https://" + self.omevv_ip
+            self.baseline_profile_url = "/omevv/GatewayService/v1/Consoles/%s/BaselineProfiles" %(self.vc_uuid)
```

Comment:
Use the omevv_api_clients model instead of direct URL in the script.

Reply:
added changes to include omevv_api_clients model